### PR TITLE
chore: downgrade lerna to resolve ESM requirement

### PR DESCRIPTION
**Pull request by michael-siek**
no qa required

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "eslint-plugin-prettier": "^5.1.3",
     "globals": "^15.2.0",
     "husky": "^9.0.10",
-    "lerna": "^8.0.1",
+    "lerna": "^8.1.3",
```
Comment (Zidious) [inline package.json:25]: Was this intended? We're not downgrading here 👀

**Review by Zidious — CHANGES_REQUESTED** [review]

File: package-lock.json [hunk 2]
```diff
@@ -3397,76 +3419,71 @@
       }
     },
     "node_modules/@lerna/create": {
-      "version": "8.1.5",
-      "resolved": "https://registry.npmjs.org/@lerna/create/-/create-8.1.5.tgz",
-      "integrity": "sha512-Ku8yTGgeumayvMr8sml72EPb6WaoJhRjMTkMZrKSJtcLNDBlDpKwyUxDxNTBNBRUYWUuJCnj7eUH7pDNuc9odQ==",
+      "version": "8.1.3",
```
Comment (michael-siek) [inline package-lock.json:3422]: @Zidious sorry the downgrade existed in the lock file.

Comment (Zidious) [reply package-lock.json:3422]: Ah missed that! Should be pin Lerna then to 8.1.3? Dependabot will update this next time around to >8.1.5

Comment (michael-siek) [reply package-lock.json:3422]: Good call just updated and pinned to 8.1.3

File: package.json [hunk 3]
```diff
@@ -22,7 +22,7 @@
     "eslint-plugin-prettier": "^5.1.3",
     "globals": "^15.2.0",
     "husky": "^9.0.10",
-    "lerna": "^8.0.1",
+    "lerna": "8.1.3",
```
Comment (Zidious) [inline package.json:25]: Will need to add it to the `dependabot.yml` too

**Review by Zidious — DISMISSED** [review]

**Review by Zidious — APPROVED** [review]

**Pull request merged**